### PR TITLE
Add batch size constraint and aggregator count constraint to the design document

### PR DESCRIPTION
**Pull request by aaomidi**
Introduces two new sections under `System constraints`

### CODE REVIEW

File: design-document.md [hunk 1]
```diff
@@ -182,6 +182,29 @@ considered in this document. For example, packet loss errors when clients
...
+the entire batch, rather than specific data points.
+
+
+### Aggregator count constraints
+
+The user of the Prio privacy preserving aggregation system may opt into using
+more than the minimum number of aggregators required for preserving privacy of
+clients. The likelihood of failures due to external factors increases
+exponentially with each additional aggregator. An example of such external
+factors are packet loss, infrastructure failure, and latency spikes.
```
Comment (acmiyaguchi) [inline design-document.md:203]: Might be useful to include an approach of how to resolve one of the issues that arises in this context (e.g., ensuring redundancy of some kind).

File: design-document.md [hunk 2]
```diff
@@ -182,6 +182,29 @@ considered in this document. For example, packet loss errors when clients
...
+
+The aggregators may introduce a maximum content size measured in bytes that the
+clients can send to the leader. This requires the client to ensure batches do
+not exceed the maximum size of any aggregator. The content size is applied to
+the entire batch, rather than specific data points.
+
+
+### Aggregator count constraints
+
+The user of the Prio privacy preserving aggregation system may opt into using
```
Comment (cjpatton) [inline design-document.md:199]:
```suggestion
The collector may opt into using
```
(Not sure if this is what you mean ... see "Terminology" above.)

**Review by cjpatton — COMMENTED** [review]
I don't think the aggregators should be able to enforce minimum batch sizes. IMO this should be a parameter chosen by the collector.

**Comment by cjpatton** [comment]
I believe the intention of this change is now subsumed by #59. Can we close this?

**Pull request closed without merging**